psvpfsparser/psvpfsparser.cpp

```diff
@@ -77,10 +77,12 @@ std::shared_ptr<IF00DKeyEncryptor> create_F00D_encryptor(const PsvPfsParserConfi
    {
       case F00DEncryptorTypes::file:
          iF00D = F00DKeyEncryptorFactory::create(cfg.f00d_enc_type, cfg.f00d_arg); 
+         break;
```

Comment:
Again. Good fix. I did not have time to test. Was planning to do further refactoring.